### ISSUE TITLE
More logs from v0.9.5.8a

### ISSUE THREAD

**Issue by BaseSquare** [megai2/d912pxy#45]
[Logs from latest alpha v0.9.5.8a.zip](https://github.com/megai2/d912pxy/files/2804164/Logs.from.latest.alpha.v0.9.5.8a.zip)

This build worked in Fullscreen 1920x1080 with no problems EXCEPT : 
Once I tried to switch from Fullscreen to simple window game crashed
Changing from Fullscreen to Windowed Fullscreen worked just fine
Changing rendering from Subsample to Native changed just fine too, but it took a while

I found out what caused the glitchy textures. Shaders in low will fuck up all of your textures, medium and above fixes it a lot. Almost completely playable honestly.

Crashes were caused only by changing the display setting Fullscreen, windowed etc. Taking a screenshot also crashes it but that is a known issue.

A lot of weird glitcyness going on all around but it's not that bad, pictures below attached. Some weird orange stuff in the distance and the water is pretty weird.

Overall FPS in Lion's Arch went from 25-30 to 33-44 averaging at 35 and FPS in Gendarran Fields, mostly empty, was 70-85 averaging at 72-74. Pretty big with High Shaders since I played with lower FPS (around 60) with Shaders on Low.

![screenshot_1](https://user-images.githubusercontent.com/37269606/51855655-746cbc00-2336-11e9-8908-30c3374f1ba4.jpg)
![screenshot_2](https://user-images.githubusercontent.com/37269606/51855657-746cbc00-2336-11e9-9b67-a7dd5183e5af.jpg)
![screenshot_3](https://user-images.githubusercontent.com/37269606/51855658-746cbc00-2336-11e9-82f3-13b3301942bc.jpg)
![screenshot_4](https://user-images.githubusercontent.com/37269606/51855659-75055280-2336-11e9-9d90-ade3180b1e94.jpg)
![screenshot_5](https://user-images.githubusercontent.com/37269606/51855660-75055280-2336-11e9-9c07-248ad74e7bdd.jpg)

Thanks for developing this, If there's something I can do to provide more data just tell me.

**Comment by megai2** [comment]
can you try it with different driver versions?

**Comment by BaseSquare** [comment]
Ok so running with different driver versions, 18.9 amd and 19.1 (latest Optional is 19.2) doesn't really fix it. I will try to play around with AMD's own settings to try and fix it.

Deleting shaders_cso.psk and running the game made it so it's the same to slightly worse. The whole effect, and switching to Medium and Low shaders made the game to not even load the textures. I would add a screenshot but when I tried to change it from simple Windowed to Windowed Fullscreen it crashed.

Will attach new logs in an hour or so.

**Comment by megai2** [comment]
If you delete that pck file, you simply delete all recompiled shaders, making d912pxy recompile them again and the last thing takes significant time, so just wait for it, if there is no textures.

**Comment by BaseSquare** [comment]
Alright one more update : 

I had 1 or 2 crashes before changing the settings below. Once after changing Borderless Fullscreen to Simply Windowed and once after clicking "Exit to Desktop" 

[Part 3 (v0.9.5.8a).zip](https://github.com/megai2/d912pxy/files/2805749/Part.3.v0.9.5.8a.zip)

One weird thing that happened while playing with the IN-GAME settings, changing Anti-Aliasing from FXAA to SMAA Low changed my screen momentarily to Black and Green outlines. Just for a moment. Weird. Nothing else occurred from changing IN-GAME settings.

I played around AMD Settings to see If I can fix it from there. No crashes occurred. 

Anti-Aliasing : Use Application Settings
Anti-Aliasing Method : Multisampling
Morphological Filtering : Off from On
Anisotropic Filtering Mode : Use Application Settings
Texture Filtering Quality : Standard
Surface Format Optimization : Off from On
Wait for Vertical Refresh :Tried On, unless application specifies from Enhanced Sync
OpenGL Triple Buffering : Off
Shader Cache : This one I tried the three settings, Off - AMD Optimized and On. No change was seen.
Tessellation Mode : Use application settings
Radeon FreeSync : Off
Display Scaling : Off
Frame Target Control : Disabled

Well no changes came from all those. If you have any other things to try I'm down, take your time though!
